src/mmw/mmw/settings/layer_settings.py

```diff
@@ -198,6 +198,32 @@
             'legend_unit_breaks': [-20, '', -8, '', 4, '', 16, '', 28, '', 40],
             'big_cz': True,
         },
+        {
+            'code': 'protected-lands-30m',
+            'display': 'Protected Lands',
+            'short_display': 'Protected Lands',
+            'css_class_prefix': 'protected-lands-30m',
+            'help_text': '',
```

Comment:
Do we know what should be the help text for this overlay?

Reply:
We don't. The help text shows up for "related layers" in the Analyze tabs. Since this isn't currently a related layer to any tab, we don't have to worry about this field.